.github/workflows/ci.yml

```diff
@@ -75,3 +68,49 @@ jobs:
       - run: pnpm add --global ember-cli yarn
       - run: pnpm vitest --testNamePattern "${{ matrix.slow-test }}"
         working-directory: tests
+
+
+  defaults_tests:
+    name: "ember-cli@${{ matrix.ember-cli }} : ${{ matrix.slow-test }}"
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ember-cli:
+          - 5.4.1
+          - 5.3.0
+
+        slow-test:
+          - defaults with npm
+          - defaults with yarn
+          - defaults with pnpm
+    steps:
+      - uses: actions/checkout@v3
+      - uses: wyvox/action-setup-pnpm@v2
+      - run: pnpm add --global ember-cli@${{ matrix.ember-cli }} yarn
+      - run: pnpm vitest --testNamePattern "${{ matrix.slow-test }}"
+        working-directory: tests
+
+  typescript_tests:
```

Comment:
typescript tests get a separate matrix, due to reasons 
(accidental non-breaking release of `@tsconfig/ember` in this case)

Reply:
(this is only needed because our tests ensure that `lint` passes across both workspaces)
(( but we don't actually have any control over the test app ))

((( which should be fine under normal circumstances )))